### ISSUE TITLE
help plz

### ISSUE THREAD

**Issue by CRANDIGAR** [AlexeyAB/Yolo_mark#122]
Ubuntu 18.04.1 LTS
After I clone the project, I just cd into the directory and follow the guide.
cmake .
make
and I got this error
--
ignacio@ignacio-GL702VSK:~/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet$ make
Scanning dependencies of target darklib
[  0%] Building CXX object CMakeFiles/darklib.dir/src/yolo_v2_class.cpp.o
[  1%] Building CXX object CMakeFiles/darklib.dir/src/activation_layer.c.o
[  1%] Building CXX object CMakeFiles/darklib.dir/src/activations.c.o
[  2%] Building CXX object CMakeFiles/darklib.dir/src/art.c.o
[  3%] Building CXX object CMakeFiles/darklib.dir/src/avgpool_layer.c.o
[  3%] Building CXX object CMakeFiles/darklib.dir/src/batchnorm_layer.c.o
[  4%] Building CXX object CMakeFiles/darklib.dir/src/blas.c.o
[  5%] Building CXX object CMakeFiles/darklib.dir/src/box.c.o
[  5%] Building CXX object CMakeFiles/darklib.dir/src/captcha.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/captcha.c: In function ‘void train_captcha(char*, char*)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/captcha.c:88:107: warning: format ‘%d’ expects argument of type ‘int’, but argument 6 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
 g, %lf seconds, %d images\n", i, loss, avg_loss, sec(clock()-time), *net.seen);
                                                                     ~~~~~~~~~^
[  6%] Building CXX object CMakeFiles/darklib.dir/src/cifar.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/cifar.c: In function ‘void train_cifar(char*, char*)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/cifar.c:36:188: warning: format ‘%d’ expects argument of type ‘int’, but argument 8 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
 .seen)/N, loss, avg_loss, get_current_rate(net), sec(clock()-time), *net.seen);
                                                                     ~~~~~~~~~^
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/cifar.c: In function ‘void train_cifar_distill(char*, char*)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/cifar.c:92:188: warning: format ‘%d’ expects argument of type ‘int’, but argument 8 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
 .seen)/N, loss, avg_loss, get_current_rate(net), sec(clock()-time), *net.seen);
                                                                     ~~~~~~~~~^
[  7%] Building CXX object CMakeFiles/darklib.dir/src/classifier.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/classifier.c: In function ‘void train_classifier(char*, char*, char*, int*, int, int, int, int, int)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/classifier.c:180:204: warning: format ‘%d’ expects argument of type ‘int’, but argument 8 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
 ages_num, loss, avg_loss, get_current_rate(net), sec(clock()-time), *net.seen);
                                                                     ~~~~~~~~~^
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/classifier.c:201:74: warning: too many arguments for format [-Wformat-extra-args]
             sprintf(buff, "%s/%s_last.weights", backup_directory, base, i);
                                                                          ^
[  7%] Building CXX object CMakeFiles/darklib.dir/src/coco.c.o
[  8%] Building CXX object CMakeFiles/darklib.dir/src/col2im.c.o
[  8%] Building CXX object CMakeFiles/darklib.dir/src/compare.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/compare.c: In function ‘void train_compare(char*, char*)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/compare.c:57:126: warning: format ‘%d’ expects argument of type ‘int’, but argument 6 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
 d images\n", (float)*net.seen/N, loss, avg_loss, sec(clock()-time), *net.seen);
                                                                     ~~~~~~~~~^
[  9%] Building CXX object CMakeFiles/darklib.dir/src/connected_layer.c.o
[ 10%] Building CXX object CMakeFiles/darklib.dir/src/convolutional_layer.c.o
[ 10%] Building CXX object CMakeFiles/darklib.dir/src/cost_layer.c.o
[ 11%] Building CXX object CMakeFiles/darklib.dir/src/cpu_gemm.c.o
[ 12%] Building CXX object CMakeFiles/darklib.dir/src/crnn_layer.c.o
[ 12%] Building CXX object CMakeFiles/darklib.dir/src/crop_layer.c.o
[ 13%] Building CXX object CMakeFiles/darklib.dir/src/cuda.c.o
[ 14%] Building CXX object CMakeFiles/darklib.dir/src/data.c.o
[ 14%] Building CXX object CMakeFiles/darklib.dir/src/deconvolutional_layer.c.o
[ 15%] Building CXX object CMakeFiles/darklib.dir/src/demo.c.o
[ 15%] Building CXX object CMakeFiles/darklib.dir/src/detection_layer.c.o
[ 16%] Building CXX object CMakeFiles/darklib.dir/src/detector.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/detector.c: In function ‘void train_detector(char*, char*, char*, int*, int, int, int, int, int)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/detector.c:329:74: warning: too many arguments for format [-Wformat-extra-args]
             sprintf(buff, "%s/%s_last.weights", backup_directory, base, i);
                                                                          ^
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/detector.c: In function ‘void calc_anchors(char*, int, int, int, int)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/detector.c:1163:62: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘double’ [-Wformat=]
                 i, box_w, box_h, anchor_w, anchor_h, best_iou);
                                                              ^
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/detector.c:1163:62: warning: format ‘%d’ expects argument of type ‘int’, but argument 4 has type ‘double’ [-Wformat=]
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/detector.c:1163:62: warning: format ‘%d’ expects argument of type ‘int’, but argument 5 has type ‘double’ [-Wformat=]
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/detector.c:1163:62: warning: format ‘%d’ expects argument of type ‘int’, but argument 6 has type ‘double’ [-Wformat=]
[ 17%] Building CXX object CMakeFiles/darklib.dir/src/dice.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/dice.c: In function ‘void train_dice(char*, char*)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/dice.c:36:107: warning: format ‘%d’ expects argument of type ‘int’, but argument 6 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
 g, %lf seconds, %d images\n", i, loss, avg_loss, sec(clock()-time), *net.seen);
                                                                     ~~~~~~~~~^
[ 17%] Building CXX object CMakeFiles/darklib.dir/src/dropout_layer.c.o
[ 18%] Building CXX object CMakeFiles/darklib.dir/src/gemm.c.o
[ 19%] Building CXX object CMakeFiles/darklib.dir/src/go.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/go.c: In function ‘void train_go(char*, char*)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/go.c:147:188: warning: format ‘%d’ expects argument of type ‘int’, but argument 8 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
 .seen)/N, loss, avg_loss, get_current_rate(net), sec(clock()-time), *net.seen);
                                                                     ~~~~~~~~~^
[ 19%] Building CXX object CMakeFiles/darklib.dir/src/gru_layer.c.o
[ 20%] Building CXX object CMakeFiles/darklib.dir/src/im2col.c.o
[ 21%] Building CXX object CMakeFiles/darklib.dir/src/image.c.o
[ 21%] Building CXX object CMakeFiles/darklib.dir/src/layer.c.o
[ 22%] Building CXX object CMakeFiles/darklib.dir/src/list.c.o
[ 22%] Building CXX object CMakeFiles/darklib.dir/src/local_layer.c.o
[ 23%] Building CXX object CMakeFiles/darklib.dir/src/lstm_layer.c.o
[ 24%] Building CXX object CMakeFiles/darklib.dir/src/matrix.c.o
[ 24%] Building CXX object CMakeFiles/darklib.dir/src/maxpool_layer.c.o
[ 25%] Building CXX object CMakeFiles/darklib.dir/src/network.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/network.c: In function ‘char* detection_to_json(detection*, int, int, char**, long long int, char*)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/network.c:731:114: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘long long int’ [-Wformat=]
 me_id\":%d, \n \"filename\":\"%s\", \n \"objects\": [ \n", frame_id, filename);
                                                                              ^
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/network.c:734:80: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘long long int’ [-Wformat=]
       sprintf(send_buf, "{\n \"frame_id\":%d, \n \"objects\": [ \n", frame_id);
                                                                              ^
[ 26%] Building CXX object CMakeFiles/darklib.dir/src/nightmare.c.o
[ 26%] Building CXX object CMakeFiles/darklib.dir/src/normalization_layer.c.o
[ 27%] Building CXX object CMakeFiles/darklib.dir/src/option_list.c.o
[ 28%] Building CXX object CMakeFiles/darklib.dir/src/parser.c.o
[ 28%] Building CXX object CMakeFiles/darklib.dir/src/region_layer.c.o
[ 29%] Building CXX object CMakeFiles/darklib.dir/src/reorg_layer.c.o
[ 29%] Building CXX object CMakeFiles/darklib.dir/src/reorg_old_layer.c.o
[ 30%] Building CXX object CMakeFiles/darklib.dir/src/rnn.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/rnn.c: In function ‘void train_char_rnn(char*, char*, char*, int, int)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/rnn.c:166:138: warning: format ‘%d’ expects argument of type ‘int’, but argument 6 has type ‘size_t {aka long unsigned int}’ [-Wformat=]
 sions = %d, text_size = %d \n", batch, steps, streams, net.subdivisions, size);
                                                                              ^
[ 31%] Building CXX object CMakeFiles/darklib.dir/src/rnn_layer.c.o
[ 31%] Building CXX object CMakeFiles/darklib.dir/src/rnn_vid.c.o
[ 32%] Building CXX object CMakeFiles/darklib.dir/src/route_layer.c.o
[ 33%] Building CXX object CMakeFiles/darklib.dir/src/shortcut_layer.c.o
[ 33%] Building CXX object CMakeFiles/darklib.dir/src/softmax_layer.c.o
[ 34%] Building CXX object CMakeFiles/darklib.dir/src/super.c.o
[ 35%] Building CXX object CMakeFiles/darklib.dir/src/swag.c.o
[ 35%] Building CXX object CMakeFiles/darklib.dir/src/tag.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/tag.c: In function ‘void train_tag(char*, char*, int)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/tag.c:67:188: warning: format ‘%d’ expects argument of type ‘int’, but argument 8 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
 .seen)/N, loss, avg_loss, get_current_rate(net), sec(clock()-time), *net.seen);
                                                                     ~~~~~~~~~^
[ 36%] Building CXX object CMakeFiles/darklib.dir/src/tree.c.o
[ 36%] Building CXX object CMakeFiles/darklib.dir/src/upsample_layer.c.o
[ 37%] Building CXX object CMakeFiles/darklib.dir/src/utils.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/utils.c: In function ‘float rand_uniform(float, float)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/utils.c:760:36: warning: integer overflow in expression [-Woverflow]
         int rnd = rand()*(RAND_MAX + 1) + rand();
                                    ^
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/utils.c:761:39: warning: integer overflow in expression [-Woverflow]
         return ((float)rnd / (RAND_MAX*RAND_MAX) * (max - min)) + min;
                                       ^
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/utils.c: In function ‘unsigned int random_gen()’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/utils.c:796:32: warning: integer overflow in expression [-Woverflow]
         rnd = rand()*(RAND_MAX + 1) + rnd;
                                ^
[ 38%] Building CXX object CMakeFiles/darklib.dir/src/voxel.c.o
[ 38%] Building CXX object CMakeFiles/darklib.dir/src/writing.c.o
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/writing.c: In function ‘void train_writing(char*, char*)’:
/home/ignacio/Escritorio/Yolo-v3 segun AlexeyAB (copia)/darknet/src/writing.c:72:188: warning: format ‘%d’ expects argument of type ‘int’, but argument 8 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
 .seen)/N, loss, avg_loss, get_current_rate(net), sec(clock()-time), *net.seen);
                                                                     ~~~~~~~~~^
[ 39%] Building CXX object CMakeFiles/darklib.dir/src/yolo.c.o
[ 40%] Building CXX object CMakeFiles/darklib.dir/src/yolo_layer.c.o
[ 40%] Building CXX object CMakeFiles/darklib.dir/src/http_stream.cpp.o
[ 41%] Building CUDA object CMakeFiles/darklib.dir/src/activation_kernels.cu.o
nvcc fatal   : Unknown option 'fopenmp'
CMakeFiles/darklib.dir/build.make:1598: recipe for target 'CMakeFiles/darklib.dir/src/activation_kernels.cu.o' failed
make[2]: *** [CMakeFiles/darklib.dir/src/activation_kernels.cu.o] Error 1
CMakeFiles/Makefile2:104: recipe for target 'CMakeFiles/darklib.dir/all' failed
make[1]: *** [CMakeFiles/darklib.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
